Document pod level resources (#48471)
kubernetes/website: content/en/docs/concepts/configuration/manage-resources-containers.md

For a particular resource, a *Pod resource request/limit* is the sum of the
resource requests/limits of that type for each container in the Pod.

> {{< note >}}

Reviewer: Don't use a `note` callout.
Try this:
```suggestion
## Pod-level resource specification
```

> <!-- feature gate PodLevelResources -->
> {{< feature-state for_k8s_version="v1.32" state="alpha" >}}

Reviewer: ```suggestion

{{< feature-state feature_gate_name="PodLevelResources" >}}

```

Starting in Kubernetes 1.32, you can also specify resource requests and limits at

> the Pod level, but only for CPU and memory. This feature is currently in alpha and

Reviewer: (nit)
```suggestion
the Pod level. At Pod level, Kubernetes {{< skew currentVersion >}} only supports resource requests
or limits for specific resource types: `cpu` and / or `memory`.
This feature is currently in alpha and
```

> requires enabling the PodLevelResources feature gate. This allows you to define an

Reviewer: ```suggestion
requires enabling the `PodLevelResources` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).

With the feature enabled, Kubernetes allows you to declare an
```

overall resource budget for the Pod, which is especially helpful when dealing with a
large number of containers where it can be difficult to accurately gauge individual
resource needs. Additionally, it enables containers within a Pod to share idle
resources with each other, improving resource utilization. 

For a pod, you can specify resource limits and requests for CPU and memory by including the following:
* `spec.resources.limits.cpu`
* `spec.resources.limits.memory`
* `spec.resources.requests.cpu`
* `spec.resources.requests.memory`

> {{< /note >}}

Reviewer: ```suggestion
```

        cpu: "500m"
```

## Pod resources example {#example-2}

> <!-- feature gate PodLevelResources -->
> {{< feature-state for_k8s_version="v1.32" state="alpha" >}}

Reviewer: ```suggestion

{{< feature-state feature_gate_name="PodLevelResources" >}}

```

This following Pod runs two containers, log-aggregator and app. The Pod has an
explicit request of 1 CPU and 128 MiB of memory, and an explicit limit of 2 CPU and
256 MiB of memory. The log-aggregator container has explicit requests and limits
set. However, the app container will simply share the resources available within the
Pod resource boundaries, as it does not have explicit requests and limits set.

```yaml
---
apiVersion: v1
kind: Pod
metadata:
  name: frontend
spec:
  resources:
    requests:
      memory: "128Mi"
      cpu: "1"
    limits:
      memory: "256Mi"
      cpu: "2"
  containers:
  - name: app
    image: images.my-company.example/app:v4
  - name: log-aggregator
    image: images.my-company.example/log-aggregator:v6
    resources:
      requests:
        memory: "64Mi"
        cpu: "250m"
      limits:
        memory: "128Mi"

>         cpu: "500m"

Reviewer: We recommend using `code_example` shortcodes for entire example manifests; see https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#source-code-files (which you are using elsewhere).

Author: yeah for this file, the other example is not using code_example shortcodes, so for consistency I kept it same. Let me fix it
